lib/gds_api/test_helpers/account_api.rb

```diff
@@ -82,6 +82,28 @@ def stub_account_api_unauthorized_user_info(**options)
         )
       end
 
+      ########################
+      # DELETE /api/oidc-users
+      ########################
+
+      def stub_account_api_success_delete_account(subject_identifier:, **options)
```

Comment:
To match the naming convention of other methods, this should be `stub_account_api_delete_user_by_subject_identifier`